testsuite/tests/lib-unix/common/utimes.ml

```diff
@@ -0,0 +1,10 @@
+let txt = "utimes.txt"
+
+let () =
+  let oc = open_out_bin txt in
+  close_out oc;
+  let t0 = 1508391026.124 in
+  Unix.utimes txt t0 t0;
+  let t1 = (Unix.stat txt).Unix.st_mtime in
+  let t2 = (Unix.stat txt).Unix.st_atime in
+  Printf.printf "t0 = %.3f\nt1 = %.3f\nt0-.t1 = %.3f\nt0-.t2 = %.3f\n%!" t0 t1 (t0 -. t1) (t0 -. t2)
```

Comment:
This test is awfully not portable and will be no end of trouble.  There is no guarantee that the file system supports sub-second resolution for file times (see FAT for a counterexample).  POSIX.1 `stat` returns file times as a `time_t` i.e. an integer number of seconds.  `utimes` may not be supported in which case `utime` is used which also uses `time_t`.  Please rewrite this test to just check that the times are what you expect up to N seconds.  Or remove the test from the testsuite.

Reply:
It's great that you always add tests, though! In addition to Xavier's portability comments, can I suggest using a different time for the two parameters (that guards coding mistakes getting parameters the wrong way around and so forth) and also testing the 0.0 case?

Reply:
@xavierleroy Good point; since I would rather have a test than not, I amended as suggested (with `N = 10`, should be enough for FAT).

@dra27 I added a test for the 0.0 case, but do not test the `atime` at all, since it is difficult to do this reliably (see comment in test file).